Kconfig.zephyr

```diff
@@ -949,6 +949,8 @@ config DEPRECATED
 	help
 	  Symbol that must be selected by a feature or module if it is
 	  considered to be deprecated.
+	  When adding this to an option, remember to follow the instructions in
+	  https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html#deprecated
```

Comment:
Could this information printed during compilation too so that developer remembers to add the information to the issue?

Reply:
The compilation warnings are not targeting the developer doing the deprecation, but rather the users affected by the deprecation. I don't think this message should be included in the compilation warning.